.github/workflows/ci_test.yml

```
@@ -144,11 +139,12 @@ jobs:
           mkdir .\data\test\runs
 
       - name: Build oxen rust project
+<<<<<<< HEAD
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)